pyproject.toml

```
@@ -79,7 +79,7 @@ argilla = ["argilla >= 2.0.0", "ipython"]
 cohere = ["cohere >= 5.2.0"]
 groq = ["groq >= 0.4.1"]
 hf-inference-endpoints = ["huggingface_hub >= 0.22.0"]
-hf-transformers = ["transformers >= 4.34.1", "torch >= 2.0.0"]
+hf-transformers = ["transformers == 4.48.3", "torch >= 2.0.0"]
```

Better to have a minimal working version, but not restricting higher ones:

```suggestion
hf-transformers = ["transformers >= 4.48.3", "torch >= 2.0.0"]
```

---

The version higher than 4.48.3 is wrong. Normally, `uv pip install` will install 4.49.0. You can refer to [this issue](https://docs.sglang.ai/start/install.html#:~:text=as%20described%20above.-,If%20you%20encounter,-ImportError%3B%20cannot) in sglang document.